### DISCUSSION TITLE
Question about SDXL safetensors to diffusers

### DISCUSSION THREAD

**Discussion by laetokang** [huggingface/diffusers#8868] [Q&A] [answered]
I'm encountering an issue when trying to fine-tune an SDXL model using DreamBooth. Here's the situation:

1. I downloaded a SafeTensors model from CivitAI.
2. I used a script to convert the single file into a multi-folder format.

 ```
import argparse
    from diffusers import StableDiffusionXLPipeline

    def parse_args():
        parser = argparse.ArgumentParser(description="Convert SDXL safetensors to Diffusers format")
        parser.add_argument("--input_file", type=str, default="/workspace/Temp/baseline.safetensors", help="Path to the input safetensors file")
        parser.add_argument("--output_dir", type=str, default="/workspace/Temp", help="Directory to save the converted model")
        return parser.parse_args()

    def main():
        args = parse_args()

        pipe = StableDiffusionXLPipeline.from_single_file(
            args.input_file, 
            use_safetensors=True,
        )

    pipe.save_pretrained(args.output_dir)
    print(f"Model successively converted and saved to {args.output_dir}")

    if __name__ == "__main__":
        main()
```

3. Upon checking the generated U-Net folder, I found that instead of a single 
- diffusion_pytorch_model.safetensors file, there were two files: 
- diffusion_pytorch_model-00001-of-00002.safetensors, diffusion_pytorch_model-00002-of-00002.safetensors

When I tried to run the DreamBooth SDXL fine-tuning script, I encountered the following error:
`An error occurred while trying to fetch /workspace/Temp: Error no file named diffusion_pytorch_model.safetensors found in directory /workspace/Temp.`

**Comment by asomoza** [comment, marked as answer]
Hi, probably the script needs to be updated to be able to load sharded checkpoints. You have a couple of solutions you can try:

- Load the downloaded model in fp16 and save it (preferably use also the "fp16" variant to save it). You will need diffusers installed from source for this since there's a fix for not sharding the models below 10GB, almost all the civitai checkpoints are in fp16.
- You can downgrade your diffusers version to 0.28.2 and save it.

**Reply by laetokang** [reply]
It works when I downgraded diffusers version to 0.28.2. 
Thank you!
